README.md

```diff
@@ -1,15 +1,225 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
```

Comment:
- [ ] This project relies on a '.env' file that is not being committed to the repo.  This means your project requires special instructions to be run correctly.  Kindly, add those instructions to your readme, so your users are able to test your project smoothly. 👍